src/ui/profile_preferences.ui

```diff
@@ -1341,124 +1341,90 @@
        <string>Mapper</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_24">
-       <item row="1" column="0">
-        <widget class="QGroupBox" name="downloadMapOptions">
+       <item row="4" column="0">
```

Comment:
Fixed `label_mapFileActionResult` size.

Just add your temporary things to the `Special options needed...`? Why would it matter where they go.

You have a point about unreadable diffs but unless we can have a script to run over them, I don't want to be ordering this stuff on every save.